packages/technical-features/kubectl-versions/src/versions.injectable.ts

```diff
@@ -0,0 +1,9 @@
+import { getInjectable } from "@ogre-tools/injectable";
+import versions from "../build/versions.json";
+
+const kubectlVersionsInjectable = getInjectable({
+  id: "kubectl-versions",
+  instantiate: (di) => versions as [string, string][],
+});
+
+export default kubectlVersionsInjectable;
```

Comment:
Why is this even an injectable?

Reply:
Maybe it shouldn't be? Okay

Reply:
One good reason for `injectable` is to control it for unit testing, even if it isn't a side-effect. Well it kinda is, for being hard-coded, volatile looking data.

Reply:
...but there's no related unit tests?